packages/web/src/components/on-ramp-button/OnRampButton.tsx

```diff
@@ -1,9 +1,9 @@
 import { OnRampProvider } from '@audius/common'
+import { IconLogoLinkByStripe as LogoStripeLink } from '@audius/harmony'
 import { Button, ButtonProps, ButtonType } from '@audius/stems'
 import cn from 'classnames'
 
-import LogoStripeLink from 'assets/img/LogoStripeLink.svg'
-import CoinbaseLogo from 'assets/img/coinbase-pay/LogoCoinbase.svg'
+import CoinbasePayLogo from 'assets/img/coinbase-pay/LogoCoinbasePay.svg'
```

Comment:
Didnt check what this looks like, but if it's one of these we probably have it in harmony already?
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/758d4ee7-d544-4f5a-8687-2ae8115432fc)
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/a3066275-05a7-478b-85ea-5e54d0b24e7b)
https://harmony.audius.co/?path=/docs/icons-logos--documentation

Reply:
unfort it is the coinbase pay logo which is different than just coinbase